### PR TITLE
fix: class Python26CompatTestCase broken by minor python versions

### PR DESCRIPTION
**Pull request by liangwen12year**
Signed-off-by: Wen Liang <liangwen12year@gmail.com>

### CODE REVIEW

File: tests/unit/test_network_connections.py [hunk 1]
```diff
@@ -60,7 +60,7 @@ class Python26CompatTestCase(unittest.TestCase):
     def assertRaisesRegex(self, exception, regex, *args, **kwargs):
         if sys.version[:3] == "2.6":
             self.assertRaises(exception, *args, **kwargs)
-        elif sys.version[:3] < "3.2":
+        elif sys.version[:3] < "3.2" and sys.version[3] == ".":
```
Comment (thom311) [inline tests/unit/test_network_connections.py:63]: why not use `sys.version_info`?

`if sys.version_info[:2] < (3,2):`

Even the doc says: "Do not extract version information out of it, rather, use version_info and the functions provided by the platform module." (https://docs.python.org/3/library/sys.html).

Comment (ffmancera) [reply tests/unit/test_network_connections.py:63]: +1

Comment (liangwen12year) [reply tests/unit/test_network_connections.py]: I updated. Thanks

**Comment by tyll** [comment]
Please explain the problem that the patch is solving in the commit message. The code looks good, thanks @thom311 for the suggestion.

**Comment by liangwen12year** [comment]
> Please explain the problem that the patch is solving in the commit message. The code looks good, thanks @thom311 for the suggestion.

Explained.

**Review by ffmancera — APPROVED** [review]

**Pull request merged**